misk-metrics/src/test/kotlin/misk/metrics/v2/FakeMetricsTest.kt

```diff
@@ -1,140 +1,151 @@
 package misk.metrics.v2
 
 import io.prometheus.client.Collector.MetricFamilySamples.Sample
+import io.prometheus.client.CollectorRegistry
 import jakarta.inject.Inject
+import misk.metrics.FakeMetricsModule
+import misk.metrics.get
+import misk.metrics.getAllSamples
```

Comment:
I'm trying to understand this approach. So if I'm writing a test, I need to remember each time to use and import this extension method instead of one with the same name on the actual `CollectorRegistry` class?

Reply:
The idea is that this class goes away, and you inject CollectorRegistry and use it. The extension methods pop up normally like other methods on the class in Intellij's auto complete.

Reply:
The collector registry doesn't have any of these methods, and I don't see a ton of value in adding our own extension just for some convenience.